lib/Extension/LanguageServerEvaluatableExpression/LanguageServerEvaluatableExpressionExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerEvaluatableExpression;
+
+use Phpactor\Container\Container;
+use Phpactor\Container\ContainerBuilder;
+use Phpactor\Extension\WorseReflection\WorseReflectionExtension;
+use Phpactor\Extension\LanguageServer\LanguageServerExtension;
+use Phpactor\Extension\LanguageServerEvaluatableExpression\Handler\EvaluatableExpressionHandler;
+use Phpactor\Container\Extension;
+use Phpactor\MapResolver\Resolver;
+
+class LanguageServerEvaluatableExpressionExtension implements Extension
+{
+    public function load(ContainerBuilder $container): void
+    {
+        $container->register('language_server_completion.handler.evaluatable_expression', function (Container $container) {
+            return new EvaluatableExpressionHandler(
+                $container->get(LanguageServerExtension::SERVICE_SESSION_WORKSPACE),
+                $container->get(WorseReflectionExtension::SERVICE_PARSER),
```

Comment:
same

Reply:
I tried `expect()`, but there was a runtime error because the `SERVICE_PARSER` was actually the cached version of the parser. I like and want it to be cached, but I wondered this was the right way to do this... Will take another look.

Reply:
the 2nd argument has no effect on the service that is returned, it doesn't even type check IIRC (although it probably should)

Reply:
Well, actually: https://github.com/phpactor/container/blob/143bbd987da798f2d7e5cedafd35b49d5051e167/lib/PhpactorContainer.php#L166 .
But I think I made a mistake with namespaces, so that's why it failed.